src/components/AssignCollectionDialog/AssignCollectionDialog.tsx

```diff
@@ -126,40 +135,59 @@ const AssignCollectionDialog: React.FC<AssignCollectionDialogProps> = props => {
           }}
         />
         <FormSpacer />
-        <ResponsiveTable>
-          <TableBody>
-            {collections &&
-              collections.map(collection => {
-                const isSelected = !!selectedCollections.find(
-                  selectedCollection => selectedCollection.id === collection.id
-                );
+        <div
```

Comment:
Stuff added in render here are exactly the same as in `AssignCategoryDetails` and almost the same as in `AssignProductDialog`, so why don't we make a component out of it?

Reply:
I guess we can merge category and collection ones